lib/Persistence/AbstractPersistence.php

```diff
@@ -88,16 +88,18 @@ protected static function _initialize()
                 throw new Exception('unable to create directory ' . self::$_path, 10);
             }
         }
-        $file = self::$_path . DIRECTORY_SEPARATOR . '.htaccess';
-        if (!is_file($file)) {
-            $writtenBytes = @file_put_contents(
-                $file,
-                'Require all denied' . PHP_EOL,
-                LOCK_EX
-            );
-            if ($writtenBytes === false || $writtenBytes < 19) {
-                throw new Exception('unable to write to file ' . $file, 11);
-            }
+        if ((strncmp("Apache", $_SERVER['SERVER_SOFTWARE'],6) == 0) or defined('_SERVER_APACHE')) {
```

Comment:
@elrido What do you say about the environmental variable in general? I'd say changing it to a config option is better…

@magikcypress If you want to see how to add a config option, in [this commit](https://github.com/PrivateBin/PrivateBin/commit/f54036976a486d3f7c83e52fd37ea357fd4e7a62) a config option was added.